--- debian/sysbench.install     1970-01-01 00:00:00.000000000 +0000
+++ debian/sysbench.install     2019-05-27 12:21:05.037733548 +0000
@@ -0,0 +1,2 @@
+usr/bin/sysbench
+usr/share/sysbench
--- debian/sysbench-tpcc.install        1970-01-01 00:00:00.000000000 +0000
+++ debian/sysbench-tpcc.install        2019-05-27 12:21:05.037733548 +0000
@@ -0,0 +1,4 @@
+tpcc.lua /usr/share/sysbench/
+tpcc_check.lua /usr/share/sysbench/
+tpcc_common.lua /usr/share/sysbench/
+tpcc_run.lua /usr/share/sysbench/
--- debian/control     2019-01-14 09:38:32.598194408 +0200
+++ debian/control     2019-01-14 09:38:30.878204231 +0200
@@ -1,7 +1,7 @@
 Source: sysbench
 Section: misc
 Priority: extra
-Maintainer: Alexey Kopytov <akopytov@gmail.com>
+Maintainer: Percona Development Team <mysql-dev@percona.com>
 Build-Depends: debhelper, autoconf, automake, libaio-dev, libtool, libmysqlclient-dev | default-libmysqlclient-dev, libpq-dev, pkg-config, python
 Standards-Version: 3.9.5
 Homepage: https://github.com/akopytov/sysbench
@@ -23,3 +23,9 @@
  - memory: a memory access benchmark
  - threads: a thread-based scheduler benchmark
  - mutex: a POSIX mutex benchmark
+
+Package: sysbench-tpcc
+Architecture: any
+Pre-Depends: sysbench (>= 1.0.0)
+Depends:  ${shlibs:Depends}, ${misc:Depends}
+Description: TPCC tests files for sysbench
--- debian/rules       2019-01-14 09:32:22.580339049 +0200
+++ debian/rules       2019-01-14 09:31:31.856639525 +0200
@@ -1,4 +1,5 @@
 #!/usr/bin/make -f
+#export DH_VERBOSE=1
 
 %:
 	dh $@
@@ -10,3 +11,13 @@
 
 override_dh_compress:
 	dh_compress -X.lua
+
+override_dh_auto_test:
+	@echo "Skipping test step"
+
+override_dh_auto_install:
+	dh_auto_install
+	cp tpcc/tpcc.lua debian/tmp/tpcc.lua
+	cp tpcc/tpcc_check.lua debian/tmp/tpcc_check.lua
+	cp tpcc/tpcc_common.lua debian/tmp/tpcc_common.lua
+	cp tpcc/tpcc_run.lua debian/tmp/tpcc_run.lua
